client/src/plugins/libparsec/definitions.ts

```diff
@@ -691,6 +707,9 @@ export interface LibParsecPlugin {
         recipient: UserID,
         role: RealmRole | null
     ): Promise<Result<null, ClientWorkspaceShareError>>
+    clientInfo(
+        client: number
+    ): Promise<Result<ClientInfo, ClientWorkspaceInfoError>>
```

Comment:
Should probably not return a `ClientWorkspaceInfoError` since it has nothing to do with a workspace.

Reply:
Done